hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), MSKTOP(i.s0.l()));
+}
```

Comment:
I was wondering if it has something to do with the safe_cast business.
I didn't want to generate a sign extension incorrectly.

     83   int8_t  b() const { return safe_cast<int8_t>(m_int); }
     84   uint8_t ub() const { return safe_cast<uint8_t>(m_int); }
     85  
hhvm/hphp/util/immed.h